core/src/test/java/io/questdb/test/cairo/mv/MatViewFuzzTest.java

```diff
@@ -202,7 +202,7 @@ public void testFullRefreshAfterUnsupportedOperations() throws Exception {
                 assertSql(
                         "count\n" +
                                 "1\n",
-                        "select count() from mat_views where view_name = '" + mvName + "' and view_status = 'valid';"
+                        "select count() from materialized_views where view_name = '" + mvName + "' and view_status = 'valid';"
```

Comment:
Please add a fuzz test with a view definition that joins the base table with the base table.

Reply:
Added a self-join fuzz test in 1e5a2871cbb7d88f9331ac7d356b7337848c4d2a